tests/10-vcdm2.js

```diff
@@ -447,15 +447,15 @@ describe('Verifiable Credentials Data Model v2.0', function() {
         await assert.rejects(issue(require(
           './input/credential-redef-type2-fail.json')));
       });
-      it2('The value of the credentialSchema property MUST be one or more' +
-        ' data schemas that provide verifiers with enough information to ' +
-        'determine if the provided data conforms to the provided schema.',
-      async function() {
+      it2('The value of the credentialSchema property MUST be one or more ' +
+        'data schemas that provide verifiers with enough information to ' +
+        'determine whether the provided data conforms to the provided ' +
+        'schema(s).', async function() {
         await issue(require('./input/credential-schema-ok.json'));
         await issue(require('./input/credential-schemas-ok.json'));
       });
       it2('Each credentialSchema MUST specify its type (for example,' +
-        ' JsonSchemaValidator2018), and an id property', async function() {
+        ' JsonSchemaValidator2018), and an id property.', async function() {
```

Comment:
Small style nit. Please indent at the end of the previous line instead of the beginning of this line.